Add command line interface and Google Colab description (#52)
Ctrl-Alt-Defeat-Foundation-Models/informatiCup2024Paper: paper.tex

As for text processors, we evaluated three different methods of augmenting generated texts. The first approach was to replace every white-space in a given text by two white-spaces. The second approach was to add some typos to the texts. This has been implemented based on the nlpaug library, which is capable of simulating certain typos as if they were made on a keyboard by a human. The typos include mistakes, like hitting a wrong key, which is close to the right one. This results in typos often created by humans. The last approach is a translator, which has been used to translate the given text into a certain language (in this case French) and re-translate it into english. As translators are not deterministic but aim to keep the semantic meaning, the resulting text should be semantically the same, but might have a different formulation.

> For the ease of use and low time consumption we implemented a simple python command line interface, that makes it easy to define a pipeline consisting of a generator, processor and evaluator. It is also possible to enter multiple processors for one pipeline, to stack and apply them together on the content.

Reviewer: I would add something like the following. The point I find is missing, is the fact that the software allows us to easily manage experiments, as we have to enter one command to run a hole experiment. 
```suggestion
For the ease of use, low time consumption and easy management of our experiments we implemented a simple python command line interface, that allows defining pipelines consisting of a generator, processor and evaluator. It is also possible to enter multiple processors for one pipeline to stack and apply them together on the content.
```

> For the ease of use and low time consumption we implemented a simple python command line interface, that makes it easy to define a pipeline consisting of a generator, processor and evaluator. It is also possible to enter multiple processors for one pipeline, to stack and apply them together on the content.

Reviewer 2: ```suggestion
For the ease of use and low time consumption we implemented a simple python command line interface, that makes it easy to define a pipeline consisting of a generator, processor and evaluator. It is also possible to enter multiple processors for one pipeline to stack and apply them together on the content.
```

> For ease of use and low time consumption we implemented a simple python command line interface, that makes it easy to define a pipeline consisting of a generator, processor and evaluator. It is also possible to enter multiple processors for one pipeline that are then stacked and applied together on the images.

Reviewer 2: ```suggestion
For the ease of use and low time consumption we implemented a simple python command line interface, that makes it easy to define a pipeline consisting of a generator, processor and evaluator. It is also possible to enter multiple processors for one pipeline, to stack and apply them together on the content.
```

> The resulting pipelines especially for the images were hardware expensive and could not be runn efficiently on our own machines. We chose to use the free google collab recources, consisting of a NVIDIA Tesla T4 GPU that would speed up the pipelines up to 40x compared to our local execution time. 

Reviewer 2: ```suggestion
The resulting pipelines, especially for the images, were very resource intensive and could not be runned efficiently on our own machines. We chose to use the free google collab recources, consisting of a NVIDIA Tesla T4 GPU that would speed up the pipelines up to 40x compared to our local execution time. 
```

> The resulting pipelines, especially for the images, were very resource intensive and could not be runned efficiently on our own machines. We chose to use the free google collab recources, consisting of a NVIDIA Tesla T4 GPU that would speed up the pipelines up to 40x compared to our local execution time. 

Reviewer 3: ```suggestion
The resulting pipelines, especially for the images, were very resource intensive and could not be runned efficiently on our own machines. We chose to use the free Google Colab recources, consisting of a NVIDIA Tesla T4 GPU that would speed up the pipelines up to 40x compared to our local execution time. 
```

Author: change in maxnets request

> The resulting pipelines, especially for the images, were very resource intensive and could not be runned efficiently on our own machines. We chose to use the free google collab recources, consisting of a NVIDIA Tesla T4 GPU that would speed up the pipelines up to 40x compared to our local execution time. 

Reviewer: Fixed two additional typos. Also I would add the aspect of having a consistent environment for the different environments.
```suggestion
The resulting pipelines, especially for the images, were very resource intensive and could not be run efficiently on our own machines. We chose to use the free Google Colab resources, consisting of a NVIDIA Tesla T4 GPU that would speed up the pipelines up to 40x compared to our local execution time and allowed to have the same environment for all experiments. 
```

> The resulting pipelines, especially for the images, were very resource intensive and could not be run efficiently on our own machines. We chose to use the free Google Colab resources, consisting of a NVIDIA Tesla T4 GPU that would speed up the pipelines up to 40x compared to our local execution time and allowed to have the same environment for all experiments. 

Reviewer 2: ```suggestion
The resulting pipelines, especially for images, were very resource intensive and could not be run efficiently on our own machines. We chose to use the free Google Colab resources, consisting of a NVIDIA Tesla T4 GPU that speeds up the pipelines up to 40x compared to our local execution time and allowed us to use the same environment for all experiments. 
```
